package.yml

```
@@ -1,5 +1,5 @@
 package: minify_images
-version: '2.0.0'
+version: '2.2.0'
```

```suggestion
version: '2.0.0'
```

Die Version besser erstmal nicht anfassen. Ein PR weiß noch nichts davon, in welchem Release er verwendet wird, sondern er bietet erstmal nur Änderungen an.